.github/workflows/run-cibuildwheel.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         run: echo "CIBW_ARCHS=${{ matrix.cibw_archs }}" >> $GITHUB_ENV
         shell: bash
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.16.2
+        uses: pypa/cibuildwheel@v2.21.2
```

Comment:
This version is both the latest as of writing, and the one that adds 3.13 support
https://github.com/pypa/cibuildwheel/releases